test/functional/legacy/eval_spec.lua

```diff
@@ -566,10 +566,10 @@ describe('eval', function()
       command("let _clipreg = ['*', getreg('*'), getregtype('*')]")
       command('let _clipopt = &cb')
       command("let &cb='unnamed'")
-      command('5y')
+      command('7y')
```

Comment:
```
[ RUN      ] eval system clipboard works: ERR
test/functional/helpers.lua:100: Vim(yank):E16: Invalid range
stack traceback:
	test/functional/helpers.lua:100: in function 'request'
	test/functional/helpers.lua:214: in function 'command'
	...build/neovim/neovim/test/functional/legacy/eval_spec.lua:569: in function <...build/neovim/neovim/test/functional/legacy/eval_spec.lua:556>
```

Reply:
If I add `eq('x', helpers.curbuf_contents())` here, I see this:

```
Passed in:
(string) 'Some first line (this text was at the top of the old test_eval.in).

Note: system clipboard is saved, changed and restored.

clipboard contents
something else'
Expected:
(string) 'x'
```

The buffer only has 6 lines, so `:7y` is indeed an error.

The Vim patch added 2 new lines to `test_eval.in`:

```
NOTE: Do not add more here, use new style test test_eval_stuff.vim

```